docs/getting-started/uwp.rst

```diff
@@ -23,13 +23,9 @@ In this walkthrough, you will build a Universal Windows Platform (UWP) applicati
      - Join
      - etc.
 
-In this article:
-	- `Prerequisites`_
-	- `Create a new project`_
-	- `Install Entity Framework`_
-	- `Create your model`_
-	- `Create your database`_
-	- `Use your model`_
+.. contents:: `In this article:`
+    :depth: 1
+    :local:
```

Comment:
Just double check this renders correctly... I think it might only give you the H1 heading of the page and you may need depth=2 to give you the actual section headings